job-server/test/spark.jobserver/JobSpecBase.scala

```diff
@@ -29,7 +31,7 @@ trait JobSpecConfig {
       "spark.jobserver.max-jobs-per-context" -> MaxJobsPerContext,
       "spark.jobserver.named-object-creation-timeout" -> "60 s",
       "akka.log-dead-letters" -> Integer.valueOf(0),
-      "spark.master" -> "local[4]",
+      "spark.master" -> "local[*]",
```

Comment:
Yes * = number of available cores, so if you have say an i7, it's equivalent to local[8]